src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -120,16 +120,18 @@ public string GetCurrentRequestHeader(ApiEvent eventInProgress)
             eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.RegionDiscovered, out string regionDiscovered);
             eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.RegionSource, out string regionSource);
             eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.IsTokenCacheSerializedKey, out string isTokenCacheSerialized);
+            eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.IsValidUserProvidedRegion, out string isValidUserProvidedRegion);
 
             // Since regional fields will only be logged in case it is opted.
-            var regionalFields = new StringBuilder();
-            regionalFields.Append(regionDiscovered);
-            regionalFields.Append(",");
-            regionalFields.Append((regionSource));
+            var platformConfig = new StringBuilder();
+            platformConfig.Append(regionDiscovered + ",");
+            platformConfig.Append(regionSource + ",");
+            platformConfig.Append(ConvertFromStringToBitwise(isTokenCacheSerialized) + ",");
+            platformConfig.Append(string.IsNullOrEmpty(isValidUserProvidedRegion) ? isValidUserProvidedRegion : ConvertFromStringToBitwise(isValidUserProvidedRegion)); //The value for this will be 1 if the region provided is valid, 0 if invalid and "" in case it could not be validated.
 
             return $"{TelemetryConstants.HttpTelemetrySchemaVersion2}" +
                 $"|{apiId},{ConvertFromStringToBitwise(forceRefresh)}" +
-                $"|{regionalFields},{ConvertFromStringToBitwise(isTokenCacheSerialized)}";
+                $"|{platformConfig}";
```

Comment:
> $"|{platformConfig}"; [](start = 15, length = 22)

can you pls. show a few examples of what this looks like from your testing? with region, without region specified, where region can be discovered, where it can't be discovered, etc?

Reply:
Added a comment with scenarios and kusto query results.